packages/utils/src/revert_error.ts

```diff
@@ -388,14 +388,16 @@ function declarationToAbi(declaration: string): RevertErrorAbi {
     const [name, args] = m.slice(1);
     const argList: string[] = _.filter(args.split(','));
     const argData: DataItem[] = _.map(argList, (a: string) => {
-        m = /^\s*([_a-z][a-z0-9_]*)\s+([_a-z][a-z0-9_]*)\s*$/i.exec(a);
+        m = /^\s*(([_a-z][a-z0-9_]*)(\[\d*\])*)\s+([_a-z][a-z0-9_]*)\s*$/i.exec(a);
```

Comment:
Nit: Could you add a comment to this regex? It's getting a bit long, and it is not immediately obvious that this regex just matches an identifier, possibly ending with an array bracket, separated from another identifier by whitespace?